Vc/sse/mask.h

```diff
@@ -222,7 +222,7 @@ template <typename T> class Mask<T, VectorAbi::Sse>
     friend reference;
     static Vc_INTRINSIC Vc_PURE value_type get(const Mask &m, int i) noexcept
     {
-        return MaskBool(m.d.m(i));
+        return m.toInt() & (1 << i);
```

Comment:
I guess that works. Looking at the previous implementation, it aliased the vector register by a T array and then accessed at `i`. That is what's happening in `m.d.m(i)`. So that might fore storing the register to memory. Your solution probably stays within the registers and might even be more efficient. AVX also uses your implementation.